src/plugins/unbounce-event-mixin.js

```diff
@@ -0,0 +1,43 @@
+import logReadQueryError from '@/util/logReadQueryError';
+import hasEverLoggedInQuery from '@/graphql/query/shared/hasEverLoggedIn.graphql';
+
+export default {
+	inject: ['apollo', 'cookieStore'],
+	created() {
+		const unbounceRoute = this.$route?.meta?.unbounce ?? false;
+		const utmEmailMedium = this.$route?.query?.utm_medium === 'email';
+
+		// Set cookie to disabled unbounce for future visits
+		if (utmEmailMedium) {
+			this.cookieStore.set('kvutm_medium_email', true);
```

Comment:
I'm not sure if our cookieStore is working properly with data setting atm but ideally this would be set for at least a year.

Reply:
I believe it will be a session cookie by default